### ISSUE TITLE
Support for multiple apps / Clarification

### ISSUE THREAD

**Issue by nunof07** [raix/push#266]
Thanks for this awesome package.

I have the following scenario and was wondering if it's possible to implement it currently or if there is any plan to support it. Basically what I am looking for is a way to do either one of the following:
1. Reconfigure Push with a different configuration.
2. Support for multiple configurations.
3. Let each individual Meteor app handle their associated notifications based on the user.

I have the following setup:
- Multiple Meteor apps running on the same server.
- All connect to the same database.
- Two separate mobile apps.
- The apps must be able to send notifications to each other.
- Each app represents a different type of user.

So basically something like this:
- A site.
- A mobile app for Android.
- A mobile app for iOS.
- B site.
- B mobile app for Android.
- B mobile app for iOS.
- C site.

And I want to be able to do this:
- A must be able to send notifications to A and B.
- B must be able to send notifications to A and B.
- C must be able to send notifications to A and B.

Currently I have this:
- Server A is setup to handle notifications for app A.
- Server B is setup to handle notifications for app B.

So I can send notifications A->A and B->B, but how to send A->B?

Ideally server A would create the notification and then server B would process it.

**Comment by raix** [comment]
you could add an endpoint for the server communication
or
use the same collection for all apps for the push collections (app + notifications)

**Comment by nunof07** [comment]
Thank you for your help.

I understand your first suggestion, but the second one I'm having trouble understanding how it would work.

Currently all apps use the same collection (they all share the same database) and each app is configured with different certificates.

If I need to send notifications to multiple devices from one of the apps, how would that work on your second suggestion?

**Comment by raix** [comment]
ah ok, at the moment this package doesn't support multiple certificates so creating an endpoint might be the best option

**Comment by nunof07** [comment]
Since all apps share the same database what I ended up doing was the following:

- Created a job queue collection.
- To send a notification insert a new document in the collection with an app ID and data for the notification.
- Each app listens for changes on the collection and filters on app ID, sending the data to `Push.send` when it finds a new document.

In this way each app can have its own configuration and we communicate between apps so that the proper one sends the notification.
